src/components/BlogList/index.jsx

```diff
@@ -67,7 +67,7 @@ const BlogList = ({
           </Button>
         </div>
       )}
-      {currentBlogs.length >= totalBlogs && currentPage > 1 && (
+      {currentBlogs.length <= totalBlogs && currentPage > 1 && (
```

Comment:
Why we are changing the condition here? 
To me the previous condition seems more accurate to render a Show Less button.